connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -2252,6 +2252,151 @@ abstract class AvroSuite
     """.stripMargin)
   }
 
+  private def checkSparkSchemaEquals(
```

Comment:
Can we also have some boundary checks (< 0, > 15)

Reply:
The value can be negative. I will add a test for > 15 case.

Reply:
I put it in the integration test to minimize code.